### PR TITLE
Feature/article

### PR DESCRIPTION
**Pull request by SaldanRoman**
https://b24-lmffg5.bitrix24.ua/workgroups/group/8/tasks/task/view/42/

### CODE REVIEW

File: css/article.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.article-links-wrapper {
+  order: 2;
+  display: flex;
+  flex-direction: column;
+}
+
+.article-links-heading {
+  font-size: 15px;
```
Comment (maradroman) [inline css/article.css]: font size must me in rem(1.07 expected)

File: css/article.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  color: #000;
+  display: block;
+  border-bottom: 1px solid #dfdfdf;
+  text-transform: uppercase;
+  margin-bottom: 26px;
+  padding-bottom: 5px;
+}
+
+.article-links {
+  font-size: 15px;
```
Comment (maradroman) [inline css/article.css]: font size must me in rem(1.07 expected)

File: css/article.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.article-content-image {
+  width: 100%;
+}
+
+@media (min-width: 800px) {
+  .article-wrapper {
+    flex-direction: row;
+  }
```
Comment (maradroman) [inline css/article.css:82]: space

File: css/article.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  .article-content {
+    width: 70%;
+    order: 2;
+  }
```
Comment (maradroman) [inline css/article.css:86]: space

File: css/article.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  .article-links-wrapper {
+    width: 30%;
+    order: 1;
+    padding: 0 20px 0 0;
+    box-sizing: border-box;
+    text-align: left;
+  }
```
Comment (maradroman) [inline css/article.css]: space

File: css/header.css [hunk 6]
```diff
@@ -536,6 +537,7 @@
 
   .header-nav-catalog-list-list {
     z-index: 1;
+
```
Comment (maradroman) [inline css/header.css]: no space

File: css/article.css [hunk 7]
```diff
@@ -0,0 +1,81 @@
+nav {
+  box-sizing: border-box;
+}
```
Comment (maradroman) [inline css/article.css]: add space

File: css/article.css [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+.nav-links-wrapper {
+  padding: 20px 0;
+}
```
Comment (maradroman) [inline css/article.css]: add space

File: css/article.css [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+    width: 30%;
+    order: 1;
+    padding: 0 20px 0 0;
+    box-sizing: border-box;
+    text-align: left;
+  }
+  .article-links-heading {
+    margin-bottom: 40px;
+  }
+}
```
Comment (maradroman) [inline css/article.css]: no brace needed?

File: articles.html [hunk 10]
```diff
@@ -0,0 +1,391 @@
...
+
+<body>
+
+    <header>
+        <div class="conteiner-stop-link"></div>
+        <section class="header-wrapper--orange">
+            <div class="header-top content-width">
+                <div class="header-top--left">
+                    <i class="fa fa-phone" aria-hidden="true"></i>
+                    <a class="header-top-text" href="tel:+18044 494 0324">044 494 0324</a>
```
Comment (maradroman) [inline articles.html]: why different numbers? in tel and inside the link

**Review by maradroman — DISMISSED** [review]
have no comments anymore.
540 - remove space
22 index.html - numbers should be identical
resolve those 2 things and merge into dev

File: articles.html (hunk 10 above)
Comment (SaldanRoman) [reply articles.html]: ???))
See how that looks in your code
and give me right nomber

**Comment by OlecsandrStelmakh** [comment]
the branch contains files that are not related to it, and which are not in dew:
css/timer.css
js/timer.js

and changes in someone else's branch (js/articles.js)

**Comment by romanivna** [comment]
resolve Olexandr's comment, removev unnecessary files from PR

File: css/header.css (hunk 6 above)
Comment (maradroman) [reply css/header.css]: remove space

File: css/timer.css [hunk 11]
```diff
@@ -0,0 +1,35 @@
+.timer {
```
Comment (maradroman) [inline css/timer.css]: merge dev and this file should be excluded from PR

File: index.html [hunk 12]
```diff
@@ -241,6 +241,35 @@
                     <button onclick="changSlide()" class="slider-buttons">&#10095</button>
                 </div>
             </div>
+    <section class="content-width">
```
Comment (maradroman) [inline index.html]: same

File: js/timer.js [hunk 13]
```diff
@@ -0,0 +1,35 @@
+const displaySeleTime = document.querySelectorAll(".timer-content-data");
```
Comment (maradroman) [inline js/timer.js]: same

File: styles.css [hunk 14]
```diff
@@ -4,7 +4,8 @@
 @import "css/product-page.css";
 @import "css/articles.css";
 @import "css/header.css";
-
+@import "css/article.css";
+@import "css/timer.css";
```
Comment (maradroman) [inline styles.css]: empty line after imports

File: js/timer.js [hunk 15]
```diff
@@ -26,22 +26,6 @@ const timerDataWrapper = document.querySelectorAll(
   if (days < 10) {
     days = "0" + days;
   }
-
-  if (displaySeleTime[0].innerText > hours) {
```
Comment (maradroman) [inline js/timer.js:30]: why is it deleted?

File: js/article.js [hunk 16]
```diff
@@ -15,25 +20,31 @@ function makeArticle() {
...
   articleContent.appendChild(title);
   articleContent.appendChild(text);
+  breadcrumbs = [{
+    name: "blog",
+    link: "blog.html"
+  }, {
+    name: articleObj[0].categorie,
+    link: "blog.html?" + articleObj[0].categorie
+  }];
+  addPointTooBreadcrumbMap(breadcrumbs);
```
Comment (maradroman) [inline js/article.js]: typo in word "Too" - shold be "To"
rename please

**Review by romanivna — APPROVED** [review]

**Pull request merged**